app/src/main/java/org/liberty/android/fantastischmemo/utils/CardTTSUtil.java

```diff
@@ -173,6 +196,18 @@ private void initTTS() {
         }  else {
             answerTTS = new NullAnyMemoTTS();
         }
+
+        if (setting.isHintAudioEnabled()){
+            String ha = setting.getHintAudio();
```

Comment:
change the "ha" variable to something more meaningful

Reply:
Lmao I know but they used "qa" (questionAudio) and "aa" (answerAudio), so I tried to keep it consistent with what they had before, hence the "ha" came about.

Reply:
Add space between ){